Sources/Datadog/RUM/RUMMonitor/Scopes/RUMApplicationScope.swift

```diff
@@ -85,7 +87,11 @@ internal class RUMApplicationScope: RUMScope, RUMContextProvider {
             startInitialSession()
         }
```

Comment:
Note, that because of:
```swift
func process(command: RUMCommand) -> Bool {
   if sessionScope == nil {
      startInitialSession()
   }
```
sending telemetry event as the very first thing in the app will be starting RUM session for the user. It can either mean starting the session earlier than it would start normally, or even starting it in the app where it won't be normally started. I wonder if this is ok 🤔?

Reply:
Yes I'll try and avoid this as well